.github/renovate.json

```diff
@@ -1,13 +1,36 @@
 {
-  "extends": ["local>brave/renovate-config"],
-  "labels": ["dependencies", "renovate"],
+  "extends": [
+    "local>brave/renovate-config"
+  ],
+  "labels": [
+    "dependencies",
+    "renovate"
+  ],
   "prConcurrentLimit": 4,
   "rangeStrategy": "pin",
   "packageRules": [
     {
-      "matchDepTypes": ["peerDependencies"],
+      "matchDepTypes": [
+        "peerDependencies"
```

Comment:
if we don't disable peerDependencies, it tries to set React to latest, which isn't compatible with brave-core (most of the time)